cromwell_tools/tests/test_utilities.py

```diff
@@ -236,6 +236,21 @@ def test_prepare_workflow_manifest_works_for_dependencies_file_with_filepath(sel
         }
         assert manifest['workflowDependencies'].getvalue() == expected_manifest['workflowDependencies'].getvalue()
 
+    def test_prepare_workflow_manifest_raises_an_error_for_dependencies_file_with_filepath_not_pointing_to_zip(self):
+        with self.assertRaises(ValueError):
+            utils.prepare_workflow_manifest(wdl_file=self.wdl_file_path,
+                                            dependencies="data/fake_test_deps.wdl")
+    
+    def test_prepare_workflow_manifest_works_for_dependencies_file_with_filepath_in_a_list(self):
+        manifest = utils.prepare_workflow_manifest(wdl_file=self.wdl_file_path,
+                                                   dependencies=[self.deps_zip_file_path])
+        expected_manifest = {
+            'workflowSource': self.wdl_file_BytesIO,
+            'workflowDependencies': self.deps_zip_file_BytesIO,
+            'workflowOnHold': 'false'
+        }
+        assert manifest['workflowDependencies'].getvalue() == expected_manifest['workflowDependencies'].getvalue()
```

Comment:
Optional suggestion: since this test is only checking `workflowDependencies` you could simplify this by doing and then remove the `expected_manifest` object: 
`assert manifest['workflowDependencies'].getvalue() == self.deps_zip_file_BytesIO`